package.json

```diff
@@ -184,6 +184,20 @@
       "type": "string",
       "value": "[0.4, 0.7, 0.1, -0.4, -0.2, -0.1]",
       "hidden": true
+    },
+    {
+      "name": "metadataRatingSystem",
+      "title": "Rate this metadata",
+      "type": "bool",
+      "value": false,
+      "hidden": true
+    },
+    {
+      "name": "metadataSource",
```

Comment:
couldn't we just pass along the URL associated? `i.e. embedly.endpoint`? why the extra pref?